static/styles/_components/_panel.scss

```diff
@@ -37,6 +37,7 @@
   background: white;
   margin-top: 3px;
   padding: .65rem 1.5rem;
+  text-align: right;
```

Comment:
This is aligning everything on the mobile panel to the right.

<img width="458" alt="screen shot 2015-08-04 at 9 42 13 am" src="https://cloud.githubusercontent.com/assets/1696495/9066551/3e814dda-3a8d-11e5-8a51-4cffc6a40c06.png">